### PR TITLE
SB3-2159: fork setup and size & positioning styling

### PR DESCRIPTION
**Pull request by rhiannanberry**
### Ticket
https://springbot.atlassian.net/browse/SB3-2159

### Changes
- added fontawesome pro
- new property type to make things easier for custom styling and possibly future validation
- setting up styles for size & pos accordion
- modifying property factory for custom labeling on the same property

### CODE REVIEW

File: src/style_manager/view/PropertyUnitValueView.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import Backbone from 'backbone';
+import PropertyView from './PropertyView';
+
+const $ = Backbone.$;
+
+export default PropertyView.extend({
+  templateLabel(model) {
+    return ``;
```
Comment (leejroberts) [inline src/style_manager/view/PropertyUnitValueView.js]: backticks not needed unless there is string interpolation happening.

File: src/style_manager/view/PropertyUnitValueView.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    const ppfx = this.ppfx;
+    const icon = this.model.get('icon');
+    if (icon) {
+      return `<div class="icon"><i class="${icon}"></i></div>`;
+    }
+    const textLabel = this.model.get('textLabel');
+    if (textLabel) {
+      return `<div class="icon"><span>${textLabel}</span></div>`;
+    }
+    return ``;
```
Comment (leejroberts) [inline src/style_manager/view/PropertyUnitValueView.js]: same

File: src/style_manager/view/PropertyUnitValueView.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  init() {
+    const model = this.model;
+    this.listenTo(model, 'change:valid', this.validityChanged);
+    this.listenTo(model, 'el:change', this.elementUpdated);
+  },
+
+  validityChanged() {
+    const ppfx = this.ppfx;
+    const fields = this.el.querySelector(`.${ppfx}fields`);
+    if (!fields) return;
```
Comment (leejroberts) [inline src/style_manager/view/PropertyUnitValueView.js]: even though art does this don't do it.
if (!fields) {
   return;
}

**Review by leejroberts — APPROVED** [review]
looks good, had three really minor comments.

**Review by riordanr1231 — COMMENTED** [review]
G00d!

**Pull request merged**